core/runtime.rs

```diff
@@ -411,6 +449,40 @@ impl JsRuntime {
     }
   }
 
+  // same as execute() but for internal JS code
+  fn execute_static(
```

Comment:
Seems superfluous, can't `execute` be used?

Reply:
It can, but I wanted to experiment with https://github.com/denoland/deno/issues/9675, which introduces `execute_static` and passes those source strings as external v8 strings.

Reply:
I want to experiment with that too, but I think first we need to support ExternalOneByteStringResource ExternalReferences. Let's remove this method for now?

Reply:
Removed for now